plugins/new-relic/retention.yaml

```diff
@@ -292,13 +328,14 @@ presetScripts:
       px.export(
         df, px.otel.Data(
           resource={
-            'service.name': df.service,
-            'k8s.container.name': df.container,
-            'service.instance.id': df.pod,
-            'k8s.pod.name': df.pod,
-            'k8s.namespace.name': df.namespace,
-            'pixie.cluster.id': df.cluster_id,
+            'service.name': df.source_service,
+            'service.instance.id': df.source_pod,
+            'k8s.namespace.name': df.source_namespace,
+            'postgres.service.name': df.destination_service,
+            'postgres.pod.name': df.destination_pod,
+            'postgres.namespace.name': df.destination_namespace,
             'k8s.cluster.name': df.cluster_name,
+            'pixie.cluster.id': df.cluster_id,
```

Comment:
CAn't coment on the specific area, but we should make sure that the attributes includes the req_body and resp_body